README.md

```diff
@@ -24,7 +24,7 @@ It has themes for B2B and B2C products under FT.com. If you're building somethin
 The header has the following features:
 
 **Required**
-- A product logo. This will default to the FT logo, but is customisable through Sass mixins.
+- A product logo. This will default to the FT logo, but is customisable through a [Sass mixin](#mixins), meaning that this can only be changed if you are using the manual build process.
```

Comment:
```suggestion
- A product logo. This will default to the FT logo, but is customisable through a [Sass mixin](#sass), meaning that this can only be changed if you are using the manual build process.
```